Source/WebCore/style/values/color/StyleDynamicRangeLimit.cpp

```diff
@@ -100,6 +101,22 @@ auto Blending<DynamicRangeLimit>::blend(const DynamicRangeLimit& from, const Dyn
     return resolve(WTFMove(function));
 }
 
+// MARK: - Flattening to platform object
+
+FlatDynamicRangeLimit DynamicRangeLimit::flatten() const
+{
+    return WTF::switchOn(*this, [&]<typename Kind>(const Kind& kind) -> FlatDynamicRangeLimit {
```

Comment:
This switch statement is written in a little weird. Is there a reason for not explicitly stating each case individually?  
```
        [](const CSS::Keyword::Standard&) {
            return FlatDynamicRangeLimit::standard();
        },
```

Reply:
From https://bugs.webkit.org/show_bug.cgi?id=278700 (and other stories) I formed the belief that `if constexpr` is better for compilation times than multiple functions.
I'm not sure it strictly applies here, but it's just a habit for me now.
Happy to change if you prefer.